stats/stats.go

```diff
@@ -56,7 +57,7 @@ func GetStatisticsSync(tempPath string, repos []string, l *log.Logger) repositor
 	var resultStats repository.Statistics
 	c := 0
 	for i := range repos {
-		stats, err := getStatsFromRepo(repos[i], tempPath, l)
+		stats, err := getStatsFromRepo(repos[i], tempPath, "", l)
```

Comment:
Why did you decide to pass empty string instead of receive the provider?

Reply:
I've not found where this function is used, so I didn't have access to the provider. Is it used anywhere?